frontend/src/views/auth/LoginView/Login.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { Formik, Form, Field } from 'formik';
+import * as Yup from 'yup';
+import styles from './Login.css';
+
+const LoginSchema = Yup.object().shape({
+  userName: Yup.string().min(4, 'Too Short!').max(50, 'Too Long!').required('Required'),
+  password: Yup.string()
+    .min(6, 'Password has to be longer than 6 characters!')
+    .required('Password is required!'),
+});
+
+export const Login = () => (
+  <div className={styles.login}>
+    <h1>Login</h1>
+    <Formik
+      initialValues={{
+        userName: '',
+        password: '',
+      }}
+      validationSchema={LoginSchema}
+      onSubmit={(values) => {
+        alert(JSON.stringify(values, null, 2));
+        console.log(values);
+      }}
+    >
+      {({ errors, touched }) => (
+        <Form>
+          <Field name="userName" placeholder="Username" />
```

Comment:
zrobiłbym osobny komponent np Input który ma już w sobie error

Reply:
Nie bardzo rozumiem jak ma to wyglądać, chcesz zrobić osobny komponent Input i jemu przesyłać dane ? 
 {({ errors, touched }) => (
        <Form>
          <Input
            userName="userName"
            type="text"
            placeholder="Username"
            isInvalid={errors.userName && touched.userName}
          />
          <Input
            userName="password"
            type="password"
            placeholder="Username"
            isInvalid={errors.password && touched.password}
          />
          <button type="submit">Login</button>
        </Form>
      )}